### PR TITLE
create impact searcher in pyserini

### PR DESCRIPTION
**Pull request by MXueguang**

### CODE REVIEW

**Comment by MXueguang** [comment]
for PR in anserini, please see https://github.com/castorini/anserini/pull/1592

File: pyserini/search/_impact_searcher.py [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+
+logger = logging.getLogger(__name__)
+
+
+# Wrappers around Anserini classes
+JImpactSearcher = autoclass('io.anserini.search.SimpleImpactSearcher')
+JImpactSearcherResult = autoclass('io.anserini.search.SimpleImpactSearcher$Result')
+
+
+class QueryEncoder:
```
Comment (lintool) [inline pyserini/search/_impact_searcher.py:39]: Are these duplications of #686 ?

File: docs/experiments-deepimpact.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# Anserini: DeepImpact for MS MARCO Passage Ranking
```
Comment (lintool) [inline docs/experiments-deepimpact.md:1]: Add a link to the Anserini docs and say something like - it's just exactly like Anserin, except you can do it from Python.

**Review by lintool — COMMENTED** [review]
A few comments...

File: pyserini/search/_impact_searcher.py (hunk 1 above)
Comment (MXueguang) [reply pyserini/search/_impact_searcher.py:39]: yes, I'll update since 686 get merged

**Review by lintool — APPROVED** [review]
lgtm

**Comment by lintool** [comment]
@MXueguang also, add links from main README?

**Pull request merged**